docs/libs/arduino-littlefs.rst

```diff
@@ -0,0 +1,22 @@
+.. _arduino_littlefs:
+
+==============
+Arduino littlefs
+==============
+
+LittleFS is a little fail-safe filesystem designed for microcontrollers and integrated in the Arduino framework 
+when used with ESP32 and ESP8266.
```

Comment:
Isn't there a generic Arduino-LittleFS implementation?

Reply:
I didn't find anything too generic for LittleFS, each time I dig into it, it's related to ESP, even if I use platformio and require `FS` + `LittleFS` it targets ESP systems.

Reply:
I see. In this case we should indicate in the name of driver that it's related to ESP only.

Reply:
wdyt of arduino_esp_littlefs?

Reply:
I buy it :slightly_smiling_face: